android/cli/commands/_build.js

```diff
@@ -878,6 +878,12 @@ AndroidBuilder.prototype.config = function config(logger, config, cli) {
 						order: 120,
 						required: true,
 						values: _t.targets
+					},
+					'sigalg': {
```

Comment:
Can you please add a `default` property with a value of `'MD5withRSA'` so that it signifies the default value in the help screen.

Reply:
Ok, good!

Reply:
Done! Thank you :)

Reply:
Oh boy, I messed up. Specifying a `default` will properly show the correct default in the help screen, but then `this.sigalg` will always have a value and it never falls back to the auto-detected `this.keystoreAlias.sigalg`.

My apologies. Please remove the `default` value.

Reply:
Oh! It's really easy thing. I have committed. Thank you for your kind review. 👍